pyproject.toml

```diff
@@ -39,6 +39,7 @@ python = "^3.7"
 importlib-metadata = {version = ">=1.7.0", python = "<3.8"}
 
 [tool.poetry.dev-dependencies]
+mypy = ">=0.931"
```

Comment:
adding mypy as a poetry dependency seems a separate concern to the other changes in this PR, no?

Reply:
just saw the above discussion around mypy. I agree it's nice to have it in the development environment, but i think perhaps that should be split out of this and into a separate PR so this one can be merged

Reply:
If a maintainer would indicate that this is what's preventing merge then I'm willing to make it separate.  

I don't know have any reason to suspect that's the case though, this project generally just seems shy of hitting the merge button.

Reply:
Yeah that's not totally unfair.
My own issue is more semantic, than logistic. They seem like genuinely separate concerns to me, and therefore belong in separate PRs.
Ultimately that's between you and the maintainers